streamz/sources.py

```diff
@@ -71,3 +72,42 @@ def start(self):
                     yield last
                 else:
                     return
+
+
```

Comment:
You can now use `@Stream.register_api(staticmethod)` here and ditch the `from_kafka` method on the Stream object (perhaps renaming this class to `from_kafka` instead)